pkg/skaffold/deploy/kustomize.go

```diff
@@ -107,3 +109,22 @@ func buildManifests(kustomization string) (io.Reader, error) {
 	}
 	return bytes.NewReader(out), nil
 }
+
+// TODO(dgageot): this code is already in KubectlDeployer
```

Comment:
why wouldn't we just add the namespace parameter to the kubectl method which we would leave as a package level utility? I'm not sure I follow why both the kustomizeDeployer and the KubeCtlDeployer need kubectl command. What is your plan here?

Reply:
I don't really have a plan. The thing is that both deployer deploy with `kubectl`. I wanted to embed `KubeCtlDeployer` into `kustomizeDeployer` but I'm not sure it's really better than a little duplication.
Another plan would be to extract this function and the state it requires into a shared package

Reply:
I extracted the kubectl CLI code to a separate package to make easier to share between both deployers.